### DISCUSSION TITLE
Airflow log table : purpose / can we purge it ?

### DISCUSSION THREAD

**Discussion by follyaka** [apache/airflow#13815] [Q&A] [answered]
Hello everyone,

I'am looking for information about the **log table** in the Airflow database schema.
In fact, in my organization we are looking for a way to improve developement in local setup. 
Regarding this, we need to get snapshot of our production database but the size the database is around 12GB and 6GB comes from log table..

So, my questions are : 
- What is the purpose of this table ?
- Is there schema documentation ?
- Could we truncate this table without any risk related to Airflow features ?

I made some test in my local setup (truncate log table), I haven't notice any broken feature, but I prefer to have a confirmation from the community to be sure.

Best regards,

Folly

**edit** : currently we are on Airflow 1.10.2 but we plan to move on Airflow 2.0.0 in this quarter.

**Comment by kaxil** [comment, marked as answer]
Yes pruning the Log table won't break anything

**Comment by follyaka** [comment]
Hello Kaxil,

Thanks a lot for your anwer.

Regards,

Folly
